### ISSUE TITLE
Hyper parameter search

### ISSUE THREAD

**Issue by EthanMarx** [ML4GW/BBHNet#169]
Now that we have an end-to-end pipeline working and initial results, we should think about adding in automated hyper-parameter tuning to allow easy determination of optimal parameters like learning rate, network size etc. 

[RayTune](https://docs.ray.io/en/latest/tune/index.html) has been mentioned as a possible option. I also recall wanting to implement our own version to allow easier interaction with condor.

**Comment by EthanMarx** [comment]
Now that we have an end-to-end pipeline working and initial results, we should think about adding in automated hyper-parameter tuning to allow easy determination of optimal parameters like learning rate, network size etc. 

[RayTune](https://docs.ray.io/en/latest/tune/index.html) has been mentioned as a possible option. I also recall wanting to implement our own version to allow easier interaction with condor.

**Comment by alecgunny** [comment]
Could be a duplicate of #36
